astropy/modeling/parameters.py

```diff
@@ -132,23 +116,29 @@ class Parameter(OrderedDescriptor):
     """
     Wraps individual parameters.
 
-    This class represents a model's parameter (in a somewhat broad sense).  It
-    acts as both a descriptor that can be assigned to a class attribute to
-    describe the parameters accepted by an individual model (this is called an
-    "unbound parameter"), or it can act as a proxy for the parameter values on
-    an individual model instance (called a "bound parameter").
+    Note: this is a new implementation of the Parameter class, and despite
+    the fact that it inherits from OrderedDescriptor, it no longer is a
+    descriptor.
```

Comment:
This sentence needs to make it more obvious in what version this changed ('new' is relative). So maybe e.g. 'Since 4.0, the Parameter class is no longer a descriptor...'

Reply:
Done